files/build/versions/host-image/versions-py3

```diff
@@ -40,5 +40,5 @@ tabulate==0.8.2
 thrift==0.13.0
 urllib3==1.26.3
 watchdog==0.10.3
-websocket-client==0.57.0
+websocket-client==0.58.0
 xmltodict==0.12.0
```

Comment:
The EOF is annoying, why it appears in some files but not others?

Reply:
I guess the file change is at the end of the file, so it shows the no new line symbol. We simply dump it by python script, all the file should be the same.